Dockerfile

```diff
@@ -12,6 +12,8 @@ FROM amazonlinux:2 as core_base
 ENV NODE_VERSION "16.x"
 ENV TERRAFORM_VERSION "1.5.3"
 ENV AWS_CLI_VERSION "2.13.25"
+ENV PYTHON_3_10_VERSION "3.10.14"
+ENV PYTHON_3_8_VERSION "3.8.16"
```

Comment:
Wouldn't it make more sense to define these in their respective sections. So the docker image only gets the `PYTHON_3_10_VERSION` variable defined if python 3.10 is actually installed?